programs/psa/crypto_examples.c

```diff
@@ -1,5 +1,18 @@
 /*
  *  Copyright (C) 2018-2019, ARM Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
```

Comment:
There should be no GPL references in the development branch!

Reply:
Oops, will correct.

Reply:
Done.